### ISSUE TITLE
Keymiss event broken on redismod:preview and redismod:latest

### ISSUE THREAD

**Issue by AlexMikhalev** [RedisGears/RedisGears#651]
Keymiss event listener is broken in both official dockers:
test_gears_keymiss.py
```
async def test_keymiss(record):
    log("called from keymiss")
    log(str(record.keys()))
    val=record['key'].split('_')
    res = test(val)
    execute('set',"works", res)
    return res 

def test(record):
    log("Called with "+ str(record))
    return record


gb = GB('KeysReader')
gb.map(test_keymiss)
gb.register(prefix='test*', commands=['get'], mode="async_local",eventTypes=['keymiss'])
```
Runing 
`gears-cli run --host 127.0.0.1 --port 6379 test_gears_keymiss.py`

redis-cli get test_1_my_command_keymiss 
nothing in the logs, batch gears work via GB.run.
Custom build ghcr.io/applied-knowledge-systems/rgcluster:edge from RedisGears master with RedisAI and no RedisJson or RedisSearch works with the same code as expected (code for docker image https://github.com/applied-knowledge-systems/the-pattern/blob/main/docker-compose.debs.yml).

```
redis-cli -p 6379 -h 127.0.0.1 INFO MODULES
# Modules
module:name=search,ver=20205,api=1,filters=0,usedby=[],using=[ReJSON],options=[handle-io-errors]
module:name=graph,ver=20601,api=1,filters=0,usedby=[],using=[],options=[]
module:name=bf,ver=20206,api=1,filters=0,usedby=[],using=[],options=[]
module:name=ReJSON,ver=20004,api=1,filters=0,usedby=[search],using=[],options=[handle-io-errors]
module:name=timeseries,ver=10410,api=1,filters=0,usedby=[],using=[],options=[]
module:name=rg,ver=10007,api=1,filters=0,usedby=[],using=[ai],options=[]
module:name=ai,ver=10003,api=1,filters=0,usedby=[rg],using=[],options=[]

# rg
rg_nexecutions:0
rg_nregistrations:1

# rg_regisrations
rg_0000000000000000000000000000000000000000-0:desc=None,reader=KeysReader,mode=async_local,numTriggered=0,numSuccess=0,numFailures=0,numAborted=0,lastError=None,regex=test*,eventTypes=[keymiss],keyTypes=None

# rg_python_stats
rg_TotalAllocated:10880526
rg_PeakAllocated:4043674
rg_CurrAllocated:3780961

# rg_python_requirements

# ai_git
ai_git_sha:7f808a934dff121e188cb76fdfcc3eb1f9ec7cbf

# ai_load_time_configs
ai_threads_per_queue:1
ai_inter_op_parallelism:0
ai_intra_op_parallelism:0
```

**Comment by MeirShpilraien** [comment]
@AlexMikhalev keymiss event is only available on master branch and in the future 1.2 version.

**Comment by AlexMikhalev** [comment]
Thank you.
A
